paullanza/home_away
# My events divisions

Added:
- Proper map markers
- My Events page divisions for attending or hosting

Fixed:
- Home page gradient
- Fonts on the event cards

Test:
- Make and join some events
- Delete some
- Verify the markers and their placements

We will still need to adjust the sizing of the boxes in the My Events page, and once Chris's code is merged the markers should be different.
